### DISCUSSION TITLE
replace ISP router

### DISCUSSION THREAD

**Discussion by 0xEDED93** [RaspAP/raspap-webgui#1048] [Q&A]
could someone point me in the right direction towards introducing these settings in the Raspap to replace my ISP router?

Username | This is your telephone number followed by  @talktalk.net. E.g. 1234567891 @talktalk.net
Password | We use automated network authentication so there is no need for a password. You can leave this blank.
VPI | 0
VCI | 38
MTU | 1432
DNS | Set as automatic (or similar)
Encapsulation | PPP over ATM (PPPoA) using VC-MUX
Modulation Type | Auto

**Comment by billz** [comment]
There is no UI to manage these settings in RaspAP. Your best bet is to try putting your ISP's router into bridge mode, connect your external device via ethernet and let RaspAP assume routing functions.

Worth noting that, due to its basic design, a Raspberry Pi or other SBC will have a fraction of the performance of a typical dedicated hardware router.

**Reply by 0xEDED93** [reply]
thank you for the reply, forgot to mention I already tried this, but my ISP´s router does not have a bridge mode.
is it possible to enter this settings without the UI ?

**Reply by billz** [reply]
Check with your ISP and ask if they support Linux.
